### ISSUE TITLE
Adding a nodeid to datasource function

### ISSUE THREAD

**Issue by Stasik0** [acplt/open62541#377]
In order to make one callback being able to handle multiple nodes.

**Comment by jpfr** [comment]
The datasource already comes with a custom handler pointer that goes into the read and write functions. So you can already reuse the same function for multiple nodes.

```c
typedef struct {
    void *handle; ///> A custom pointer to reuse the same datasource functions for multiple sources
    UA_StatusCode (*read)(void *handle, UA_Boolean includeSourceTimeStamp, const UA_NumericRange *range, UA_DataValue *value);
    UA_StatusCode (*write)(void *handle, const UA_Variant *data, const UA_NumericRange *range);
} UA_DataSource;
```

**Comment by Stasik0** [comment]
i am still not convinced, but okay, i will come up with it again, when i get a real use-case

**Comment by jpfr** [comment]
This was implemented and merge in 705f03b.
